src/core/Akka/Actor/ActorPath.cs

```diff
@@ -254,7 +254,8 @@ public bool Equals(ActorPath other)
             var a = path;
             foreach (string element in name)
             {
-                a = a / element;
+                if(!string.IsNullOrEmpty(element))
+                    a = a / element;
```

Comment:
I've also added fix for ActorPath operator / used in singleton proxy. This way it's also handled in akka